src/components/NanoContract/NanoContractsList.component.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
```

Comment:
Can you please remove the `.component.` from those components?

Reply:
Done